sources/operator.1.14.0.psmdb-operator.json

```diff
@@ -0,0 +1,90 @@
+            "critical": false
+          },
+          "5.0.11-10": {
+            "image_path": "percona/percona-server-mongodb:5.0.11-10",
+            "image_hash": "sha256:da3713525d76a354435e1ab8fda12a06407e7eca8b8e72b9ac0163a34c8eb735",
+            "status": "recommended",
+            "critical": false
+          },
+          "5.0.7-6": {
+            "image_path": "percona/percona-server-mongodb:5.0.7-6",
+            "image_hash": "3f4849a17236c3849a513f46caa39fbc6da0414f98d27e080fbe0496fa9e86a2",
+            "status": "available",
+            "critical": false
+          },
+          "5.0.4-3": {
+            "image_path": "percona/percona-server-mongodb:5.0.4-3",
+            "image_hash": "4ac4cff1dac52ea109e9a68a61de44c75b62292bb4676cf8efd1e00000d8adf3",
+            "status": "available",
+            "critical": false
+          },
+          "4.4.16-16": {
+            "image_path": "percona/percona-server-mongodb:4.4.16-16",
+            "image_hash": "sha256:402b5e5b08ac73c74a47c72d002251a086f9ad28b0594fbae5c34757b294ce13",
+            "status": "recommended",
+            "critical": false
+          },
+          "4.4.13-13": {
+            "image_path": "percona/percona-server-mongodb:4.4.13-13",
+            "image_hash": "059c3c9a0360d6823905e39b52bdcaf76c3929c93408c537f139cee835c2bc0f",
+            "status": "available",
+            "critical": false
+          },
+          "4.4.10-11": {
+            "image_path": "percona/percona-server-mongodb:4.4.10-11",
+            "image_hash": "ea73a506fa02604660e3ef7d452d142a89587bb5daca15d3cc1b539a9b1000c5",
+            "status": "available",
+            "critical": false
+          },
+          "4.4.8-9": {
+            "image_path": "percona/percona-server-mongodb:4.4.8-9",
+            "image_hash": "4d29b3557c949f95009eaccf7a8f56215ac609406d230be87b6eaa072e0c1f69",
+            "status": "available",
+            "critical": false
+          },
+          "4.2.22-22": {
+            "image_path": "percona/percona-server-mongodb:4.2.22-22",
+            "image_hash": "sha256:da4634df780563e10a547662c58c8ce28fbe5c98e1ac8b42b4f6be87f292e92b",
+            "status": "recommended",
+            "critical": false
+          }
+        },
+        "pmm": {
+          "2.30.0": {
+            "image_path": "percona/pmm-client:2.30.0",
+            "image_hash": "sha256:de556410de32a49a8a6bc157536881e2baefc8549a1094d6c2c70242a3c792cb",
+            "status": "recommended",
+            "critical": false
+          }
+        },
+        "backup": {
+          "2.0.3": {
+            "image_path": "percona/percona-backup-mongodb:2.0.3",
+            "image_hash": "sha256:d0f4515fe63652f17d2f45f1d865b4715af06f7b52e5e3f0fd6d23d0a2db9e2b",
+            "status": "recommended",
+            "critical": false
+          }
+        },
+        "operator": {
+          "1.14.0": {
+            "image_path": "percona/percona-server-mongodb-operator:1.14.0",
```

Comment:
@tplavcic @nmarukovich how do we handle this case? I need VS updated so upgrade tests can pass, but we don't have new operator images built yet.

Reply:
I believe for testing we can use image  perconalab/percona-server-mongodb-operator:1.14.0
You can rebuild if you want.

Reply:
@tplavcic wdyt what about the above?

Reply:
I propose we just set `perconalab/percona-server-mongodb-operator:main` for now. We will not need to rebuild it and it will be more correct for the test, but as I mentioned I don't believe this is used anywhere in the operator itself.